src/dune_rules/dep_rules.ml

```diff
@@ -1,6 +1,8 @@
 open Import
 open Memo.O
-open Ocamldep.Modules_data
+open Dep_gen.Modules_data
+
+module Current_dep_gen : Dep_gen.S = Codept (* TODO: make dynamic *)
```

Comment:
I'm guessing the choice of using codept instead of ocamldep should be controlled by an `(using codept 0.1)` stanza. Although I'm not sure how to get access to the project around here in order to use `Dune_project.is_extension_set` or something.

Reply:
Passing it via `Ocamldep.Modules.t` seems like a reasonable option.

Reply:
Oh yeah, that's quite direct. I initially thought it'd have to be passed around through a lot more places.
Anyway, I now added the extension.